### PR TITLE
Add what's expected of the Solutions Engineer on support

### PR DESCRIPTION
**Pull request by LolloneS**
### What is needed from the reviewers?

A 👍 

### Why is this piece of content here?

- too rudimental and unstructured for our docs
- not secret so no need to go to intranet

Closes https://github.com/giantswarm/giantswarm/issues/27053

### CODE REVIEW

File: content/docs/support-and-ops/support/support-FAQ.md [hunk 1]
```diff
@@ -22,4 +22,14 @@ A good way to do this is to gather enough information to answer all the followin
 - Did anything change in the cluster recently?
 - Is the issue a real issue, or is it just a question about how to do something or a feature request? If it is not an issue, we can move the discussion to a GitHub issue.
 
+## What's expected of the on-support engineer?
 
+Out of a Solutions Engineer on support, it is expected that:
```
Comment (lyind) [inline content/docs/support-and-ops/support/support-FAQ.md:27]:
```suggestion
A Solutions Engineer on support shall:
```

File: content/docs/support-and-ops/support/support-FAQ.md [hunk 2]
```diff
@@ -22,4 +22,14 @@ A good way to do this is to gather enough information to answer all the followin
...
+
+- the person qualifies the request by following the [Support Template](#support-template-qualifying-a-request)
```
Comment (lyind) [inline content/docs/support-and-ops/support/support-FAQ.md:29]:
```suggestion
- qualify the request by following the [Support Template](#support-template-qualifying-a-request)
```

File: content/docs/support-and-ops/support/support-FAQ.md [hunk 3]
```diff
@@ -22,4 +22,14 @@ A good way to do this is to gather enough information to answer all the followin
...
+- the person tries to perform some basic debugging according to opsrecipes/FAQ entries (look at pods logs, check if nodes are Ready, ...)
```
Comment (lyind) [inline content/docs/support-and-ops/support/support-FAQ.md:30]:
```suggestion
- try to perform basic debugging according to opsrecipes/FAQ entries (look at pods logs, check if nodes are Ready, ...)
```

File: content/docs/support-and-ops/support/support-FAQ.md [hunk 4]
```diff
@@ -22,4 +22,14 @@ A good way to do this is to gather enough information to answer all the followin
...
+- the person drives communication with the customer. This will allow us to have a more diverse face to the customer and not always the same people solving problems even though they might have helped in the background.
```
Comment (lyind) [inline content/docs/support-and-ops/support/support-FAQ.md:31]: And so on...

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**